src/base/collect_tags.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+
+import argparse
+import glob
+import os
+import re
+import shutil
+import subprocess
+
+
+def parse_args():
+    """Parse command line arguments and return them."""
+    ap = argparse.ArgumentParser(description='collect debug and trace tags')
+    ap.add_argument('destdir', help='where to put the results')
+    ap.add_argument('type', choices=['Debug', 'Trace'], help='which tags to collect')
+    ap.add_argument('basedir', help='where to look for source file')
+    return ap.parse_args()
+
+
+def collect_tags(basedir: str):
```

Comment:
Not sure if we should use type hints since this would require a minimum version of Python 3.5.